metricbeat/docker-compose.yml

```diff
@@ -49,11 +49,11 @@ services:
 
   # Used by base tests
   metricbeat:
-    image: docker.elastic.co/integrations-ci/beats-metricbeat:${BEAT_VERSION:-7.13.4}-1
+    image: docker.elastic.co/integrations-ci/beats-metricbeat:${BEAT_VERSION:-7.14.0}-1
```

Comment:
Somehow it states here 7.14.0 but I would argue this should likely be 7.16.0 for 7.x branch? @andresrc

Reply:
For this it uses the latest released version it seems, which at that point was 7.14, not sure if we should move to 7.15

Reply:
++ on using 7.15.0 now as the "fallback".